app/controllers/summary_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SummaryController < ApplicationController
+  before_action :authorize
+
+  def index
+    client_namespace = current_user.client_slug.titleize
+    @client_summarizer = ClientSummarizer.new(
+      client_namespace: client_namespace,
+      fiscal_year: params[:fiscal_year]
+    )
+    @summary = @client_summarizer.run
+  end
+
+  private
+
+  def authorize
```

Comment:
in what cases do we define this here vs writing a pundit policy? Don't mind this being here, just wondering